packages/extension-points/package.json

```diff
@@ -1,11 +1,14 @@
 {
-  "name": "@drupal/extension-points-bundler",
+  "name": "extension-points",
```

Comment:
If  we don't have that, ```yarn workspaces ... prettier``` doesn't work.

Reply:
What happens when we want to publish this to npm? Also we are dropping the `bundler` part of this project name?

Reply:
Good point, I'll  revert it